openjre.hcl

```diff
@@ -26,31 +26,66 @@ platform "amd64" {
   }
 }
 
-version "11.0.10_9" {
```

Comment:
Isn't this getting rid of this version? We can't remove versions.

Reply:
It looks like it's missing linux/arm64 tarballs (which weren't produced at that time) but that the others are present:

```
🐚 ~/dev/hermit-packages $ hermit install openjre-11.0.10_9
info:openjre-11.0.10_9:install: Installing openjre-11.0.10_9
🐚 ~/dev/hermit-packages $ java -version
openjdk version "11.0.10" 2021-01-19 LTS
OpenJDK Runtime Environment Zulu11.45+27-CA (build 11.0.10+9-LTS)
OpenJDK 64-Bit Server VM Zulu11.45+27-CA (build 11.0.10+9-LTS, mixed mode)
```